internal/admin/dashboard/static/fonts/inter.css

```
@@ -0,0 +1,18 @@
+/* Inter (variable, latin + latin-ext subsets) — vendored from Google Fonts for offline use.
+   v20 ships one variable woff2 per subset, so a single @font-face covers weights 400-700. */
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)